### PR TITLE
Awin affiliate tracking

### PR DESCRIPTION
**Pull request by rupertbates**
We are launching an affiliate scheme to pay commission to referrers when a we sell a product.

We are working with a company called Awin to do this.

[Trello card here](https://trello.com/c/pbQRuFKA/1117-affiliate-tags)

[Implementation docs here](https://drive.google.com/drive/folders/19AOGPkPbbFjqPJTdCcCba3EwpLiSHBwW)

*NB. I am in the process of checking the data we are sending to Awin with the GDPR team*

### CODE REVIEW

File: app/model/SubscriptionOps.scala [hunk 1]
```diff
@@ -54,6 +54,7 @@ object SubscriptionOps extends LazyLogging {
       }
     }
 
+    def firstPrice = firstPlan.charges.price.getPrice(currency).map(_.amount).getOrElse(0F)
```
Comment (paulbrown1982) [inline app/model/SubscriptionOps.scala]: As you're adding this to a string in the JSON in thankyou.scala.html, I think you can do .map(_.pretty).mkString

File: app/views/main.scala.html [hunk 2]
```diff
@@ -49,5 +49,7 @@
 
         @fragments.global.footer(product, edition)
         @fragments.footerJavaScript()
+        <!--AWIN affiliate tracking tag - see here: https://drive.google.com/drive/folders/19AOGPkPbbFjqPJTdCcCba3EwpLiSHBwW -->
+         <script src="https://www.dwin1.com/11152.js" type="text/javascript" defer="defer"></script>
```
Comment (paulbrown1982) [inline app/views/main.scala.html]: You could add the script to the DOM inside the init of awin.js, then if the user has opted out via DNT, or we're in test mode and so analyticsEnabled() returns false,  we won't load the JS from dwin1.com

**Comment by johnduffell** [comment]
is this PR still needed?

**Comment by johnduffell** [comment]
>28 days old, reopen/open another if needed

**Pull request closed without merging**